hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMultiMapProxy.java

```diff
@@ -148,6 +149,12 @@ public boolean remove(Object key, Object value) {
         MultiMapRemoveCodec.ResponseParameters resultParameters = MultiMapRemoveCodec.decodeResponse(response);
         return new UnmodifiableLazyList<V>(resultParameters.response, getSerializationService());
     }
+    public void delete(Object key) {
+        checkNotNull(key, NULL_KEY_IS_NOT_ALLOWED);
+
+        Data keyData = toData(key);
+        MultiMapDeleteCodec.encodeRequest(name, keyData, 4L);
```

Comment:
missing the actual invocation here eg `ClientMessage response = invoke(request, keyData);`

Also, for the client-side implementation you need to:
- create a message task on the member side that will handle the invocation from the client (eg look at `MultiMapRemoveMessageTask` which implements `MultiMap.remove(key)`).
- register that message task in `DefaultMessageTaskFactoryProvider`

Reply:
Alright, I added the RemoveMessageTask and registered it.